staging/src/k8s.io/legacy-cloud-providers/azure/azure_vmss.go

```diff
@@ -40,6 +40,9 @@ import (
 )
 
 var (
+	// virtualMachineScaleSetsDeallocating indicates VMSS instances are in Deallocating state.
+	virtualMachineScaleSetsDeallocating = "Deallocating"
```

Comment:
why adding this const, it's not used in current code

Reply:
it's moving from azure_client.go to azure_vmss.go, and it is used in `ensureVMSSInPool()`